packages/edit-navigation/src/components/header/style.scss

```diff
@@ -28,3 +28,8 @@
 .edit-navigation-header__add-menu {
 	min-width: 220px;
 }
+
+.edit-navigation-header__menu-selection-dropdown {
+	// Appear below the modal overlay.
+	z-index: 99998;
```

Comment:
shouldn't we use a variable here (added to _z-index.scss)?

Reply:
Done in a1dc2f3.